content/docs/user-guide/testing/quality/_index.md

```diff
@@ -0,0 +1,216 @@
+
+
+## Quality levels 
+
+Quality levels can be defined in Settings Registry JSON files within each quality group JSON object.  
+
+Quality levels within each group JSON object should be defined in the following format.
+```json
+{
+    "O3DE": {
+        "Quality": {
+            "Groups": {
+                "q_general": {
+                    "Levels": [ // quality level names should be a single word
+                        "Low",
+                        "Medium",
+                        "High",
+                        "VeryHigh"
+                        ...
+                    ]
+                }
+            }
+        }
+    }
+}
+```
+
+Each group can have it's own quality levels defined with custom names and number.  Providing quality level names other than `Low`, `High`, etc. can be useful when defining levels for settings that aren't necessarily better or worse, for example you may have settings for power consumption levels like `HighPerformance`, `Balanced`, `EnergyEfficient`. 
+
+## Customizing quality levels for your project
+
+To override and customize the quality settings in the engine and the gems your project uses, define your own `.setreg` file in your projects `Registry` folder that contains the appropriate JSON content.  For example, if you wanted to override the default quality level to use in your game to be `VeryHigh` and provide a specific shadow console variable for a specific quality level you could add the following `.setreg` file to your project.
+
+### Project/Registry/quality.setreg
+
+```json
+{
+    "O3DE": {
+        "Quality": {
+            "Groups": {
+                "q_general": {
+                    "Default": 3 // change the default level to use to be VeryHigh instead of High
+                },
+                "q_shadows": {
+                    "Settings": {
+                        // example console variable setting for each of the 4 quality
+                        // levels in q_shadows (Low, Medium, High, VeryHigh)
+                        "r_shadowResolution": [128, 512, 1024, 4096]
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+## Platform specific quality settings
+
+Quality settings can easily be customized for specific platforms because of the existing platform abstraction built into the Settings Registry.  By placing quality `.setreg` files in the appropriate `Registry/Platform/<platform name>` folder they will be used for that specific platform and override any settings in the `Registry/` folder.
+
+For example, to change the default quality level to use on the Android platform you could add the following `.setreg` file to your project.
+
+### Project/Registry/Platform/Android/quality.setreg
+
+```json
+{
+    "O3DE": {
+        "Quality": {
+            "Groups": {
+                "q_general": {
+                    "Default": 0 // change the default level to use to be Low 
```

Comment:
nit:
`// change the default level to use to be Low`
to
`// change the default level to Low`